src/Domain/Creators/Tags/ObjectCenter.hpp

```diff
@@ -24,23 +24,11 @@ struct Grid;
 namespace domain::Tags {
 /// \ingroup DataBoxTagsGroup
 /// \ingroup ComputationalDomainGroup
-/// Base tag to retrieve the grid frame centers of objects in the domain
-/// corresponding to the `ObjectLabel`.
+/// The grid frame center of the given object.
```

Comment:
Mention in the docs that this requires the domain creator to have a grid anchor named `CenterA/B`